build.xml

```diff
@@ -121,6 +122,12 @@
   <target name="-real-build">
     <mkdir dir="${build-src.dir}"/>
 
+    <available property="compile.java.isNewer"
+      classname="${compile.java.newerClassname}"
+      classpath="${compile.java.bootclasspath}" ignoresystemclasses="true"/>
+    <fail if="compile.java.isNewer"
+      message="build.bootclasspath is newer than JDK ${compile.java.version}."/>
```

Comment:
Is this condition bad?  Is this condition good? What can someone do about it?

Reply:
Yeah, it's bad, hence the build fail. I'm trying to fix this (documented) error:

If you specify the wrong JDK version in build.bootclasspath, the potential errors are very difficult to detect. The class file versions will be correct, but the code may not run reliably on the desired version of the JVM, and the failures may happen at any time.